fvm/systemcontracts/system_contracts.go

```diff
@@ -65,6 +65,8 @@ const (
 	FlowTokenAccountIndex     = 3
 	FlowFeesAccountIndex      = 4
 	EVMStorageAccountIndex    = 5
+
+	LastSystemAccountIndex = EVMStorageAccountIndex
```

Comment:
I suggest adding additional documentation here.

These are the account indexes of system contracts **as deployed by the default bootstrapping**.

However, this package also contains information about chains which were not deployed by the default bootstrapping, like Mainnet and Testnet. For those chains, I don't think these indices will point to the right contracts.